lib/domain/rss_item_itunes.dart

```diff
@@ -38,31 +38,31 @@ class RssItemItunes {
 
   factory RssItemItunes.parse(XmlElement element) {
     final episodeStr =
-        findElementOrNull(element, 'itunes:episode')?.text.trim();
-    final seasonStr = findElementOrNull(element, 'itunes:season')?.text.trim();
+        findElementOrNull(element, 'itunes:episode')?.innerText.trim();
+    final seasonStr = findElementOrNull(element, 'itunes:season')?.innerText.trim();
     final durationStr =
-        findElementOrNull(element, 'itunes:duration')?.text.trim();
+        findElementOrNull(element, 'itunes:duration')?.innerText.trim();
 
     return RssItemItunes(
-      title: findElementOrNull(element, 'itunes:title')?.text.trim(),
+      title: findElementOrNull(element, 'itunes:title')?.innerText.trim(),
       episode: episodeStr == null ? null : int.tryParse(episodeStr),
       season: seasonStr == null ? null : int.tryParse(seasonStr),
       duration: durationStr == null ? null : parseDuration(durationStr),
       episodeType: newRssItunesEpisodeType(
-          findElementOrNull(element, 'itunes:episodeType')),
-      author: findElementOrNull(element, 'itunes:author')?.text.trim(),
-      summary: findElementOrNull(element, 'itunes:summary')?.text.trim(),
+        findElementOrNull(element, 'itunes:episodeType'),),
+      author: findElementOrNull(element, 'itunes:author')?.innerText.trim(),
+      summary: findElementOrNull(element, 'itunes:summary')?.innerText.trim(),
       explicit: parseBoolLiteral(element, 'itunes:explicit'),
-      subtitle: findElementOrNull(element, 'itunes:subtitle')?.text.trim(),
+      subtitle: findElementOrNull(element, 'itunes:subtitle')?.innerText.trim(),
       keywords: findElementOrNull(element, 'itunes:keywords')
-              ?.text
+              ?.innerText
               .split(',')
               .map((keyword) => keyword.trim())
               .toList() ??
           const <String>[],
       image: RssItunesImage.parse(findElementOrNull(element, 'itunes:image')),
       category: RssItunesCategory.parse(
-          findElementOrNull(element, 'itunes:category')),
+        findElementOrNull(element, 'itunes:category'),),
```

Comment:
Looks like there's a formatting issue here - looks like this comma should be followed by a line break, or if you don't want a line break, then there shouldn't be a comma.